src/compile/legend/parse.ts

```diff
@@ -172,6 +172,17 @@ export function mergeLegendComponent(mergedLegend: LegendComponent, childLegend:
     // Cannot merge due to inconsistent orient
     return undefined;
   }
+
+  const mergedDirection = mergedLegend.getWithExplicit('direction');
```

Comment:
Is this really necessary (and the one above) since we could just use the properties of the first legend instead.